muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/index.tsx

```diff
@@ -3,199 +3,246 @@
 //it does not have the same capabilities and efficiency as the other components, and cannot be easily modified
 //please remove it
 
-import * as React from 'react';
-
-import mApi from '~/lib/mApi';
-import { WorkspaceType, MaterialContentNodeType, MaterialCompositeRepliesType } from '~/reducers/workspaces';
-import promisify from '~/util/promisify';
-
-import { i18nType } from '~/reducers/base/i18n';
-import { StatusType } from '~/reducers/base/status';
-import { StateType } from '~/reducers';
-import { Dispatch, connect } from 'react-redux';
-import { WebsocketStateType } from '~/reducers/util/websocket';
-import { bindActionCreators } from 'redux';
-import { UpdateAssignmentStateTriggerType, updateAssignmentState,
-  setWorkspaceMaterialEditorState, SetWorkspaceMaterialEditorStateTriggerType,
-  UpdateWorkspaceMaterialContentNodeTriggerType, updateWorkspaceMaterialContentNode,
-  requestWorkspaceMaterialContentNodeAttachments, RequestWorkspaceMaterialContentNodeAttachmentsTriggerType } from '~/actions/workspaces';
-import { DisplayNotificationTriggerType, displayNotification } from '~/actions/base/notifications';
-
-import '~/sass/elements/rich-text.scss';
-import '~/sass/elements/material-page.scss';
+import * as React from "react";
```

Comment:
Is this just prettier?...

I can't get to find what changed, and I am afraid this could cause merge conflicts with other PRs

Reply:
Mostly prettier changes. Allthough I changed documentation marking in props because I was annoyed when there where documentation that code editor didn't info me as it could have help me to understand why and what for those props are.